doc/grpcSupport.md

```diff
@@ -0,0 +1,25 @@
+# gRPC support
+
+Experimental feature - it's not complete and API may change
+
+tAPIr supports currently defining gRPC endpoints and generating proto files from these definitions.
+## Defining endpoints
+The definition of an endpoint requires names for service and method. We can simply pass them as an input path the endpoint definitions e.g. `endpoint.in("Library" / "AddBook")`.
```

Comment:
you mean for an grpc endpoint? is this a tapir endpoint you are talking about? it's not clear